modules/mcuboot/CMakeLists.txt

```diff
@@ -489,13 +489,27 @@ if(CONFIG_BOOTLOADER_MCUBOOT)
         )
     else()
       # No network core update
-      set(generate_bin_files
-        ${PROJECT_BINARY_DIR}/${app_core_binary_name}
-        )
-      set(generate_script_params
-        "load_address=$<TARGET_PROPERTY:partition_manager,PM_APP_ADDRESS>"
-        "version_MCUBOOT=${CONFIG_MCUBOOT_IMAGE_VERSION}"
-        )
+      if(CONFIG_BOOT_BUILD_DIRECT_XIP_VARIANT)
+        set(secondary_app_core_binary_name mcuboot_secondary_app_update.bin)
```

Comment:
Should we enclose `mcuboot_secondary_app_update.bin` in double quotes?

Reply:
As far as i know it is not needed, here i aligned how it was done for example here: `set(app_core_binary_name app_update.bin)`